src/partials/auth-security.md

```diff
@@ -24,4 +24,10 @@ If an user is imported into Appwrite with hash differnt than Argon2, the passwor
 Encourage passwords that are hard to guess by disallowing users to pick passwords that contain personal data.
 Personal data includes the user's name, email, and phone number.
 
-Disallowing personal data can be enabled in the Auth service's **Security** tab on the Appwrite Console.
\ No newline at end of file
+Disallowing personal data can be enabled in the Auth service's **Security** tab on the Appwrite Console.
+
+# Session alerts {% #session-alerts %}
+
+Set up email alerts for your users so that whenever another session is created outside of their existing session using their user account information, such as their email address, they will be alerted to the new session.
+
+Users must have a valid email address to receive email notifications. This isn't the case when using [Magic URL](/docs/products/auth/magic-url) login, [Email OTP](/docs/products/auth/email-otp) login, or [OAuth2](/docs/products/auth/oauth2) login. To toggle session alerts, navigate to **Auth** > **Security** > Session Alerts.
```

Comment:
> Users must have a valid email address to receive email notifications. This isn't the case when using

This makes it sound like Magic URL, Email OTP, and OAuth2 are not cases with valid emails.

> To toggle session alerts, navigate to **Auth** > **Security** > Session Alerts.
Comment

Let's move this to a separate paragraph because it's a different topic.